TTT/Game.cpp

```diff
@@ -0,0 +1,90 @@
+#include "Game.hpp"
+
+Game::Game() {
+	ticTacToe();
+}
+
+Game::~Game() {};
+
+void Game::setGameMode() {
```

Comment:
I think that something like queryGameMode would be more appropriate, set* methods usually accept the value to set as an argument.